conda/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: scwidgets
+  version: "0.1"
+
+source:
+  path: ..
+
+requirements:
+  run:
+    - python {{ python }}
+    - scipp
+    - ipywidgets
+
+test:
+  import:
+    - scwidgets
+  requires:
+    - pytest
+  source_files:
+    - .
+  commands:
+    - python -m pytest -v
+
```

Comment:
I would recommend adding a `build` section with `number` (see my PR) so that we can increment with versions since tag.